app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorStatusCalculator.kt

```diff
@@ -4,35 +4,39 @@ import com.f2prateek.rx.preferences2.Preference
 import io.reactivex.schedulers.Schedulers.io
 import org.simple.clinic.di.AppScope
 import org.simple.clinic.sync.DataSync
+import org.simple.clinic.sync.LastSyncedState
 import org.simple.clinic.sync.SyncGroup
 import org.simple.clinic.sync.SyncProgress
 import org.simple.clinic.util.Just
-import org.simple.clinic.util.Optional
 import org.threeten.bp.Clock
 import org.threeten.bp.Instant
 import javax.inject.Inject
-import javax.inject.Named
 
 @AppScope
 class SyncIndicatorStatusCalculator @Inject constructor(
     dataSync: DataSync,
     clock: Clock,
```

Comment:
We've replaced `Clock` with `UtcClock` and `UserClock`. I just realized that Dagger will throw an error once you start using `SyncIndicatorStatusCalculator` somewhere.